### ISSUE TITLE
Clean console.log() outputs from npm package

### ISSUE THREAD

**Issue by aboglioli** [basvandenberg/angular2-select#64]
NPM package has console.log() outputs.

If I build angular2-select from this repository (tag `v1.0.0-beta.0`), the generated code (in `dist/`) doesn't have any console.log() unlike code from NPM.

I'm having troubles with my unit tests (using karma) because there are a lot of irrelevant outputs .

Reproduce in a clean folder:
```bash
echo "{}" > package.json
npm install angular2-select
cat node_modules/angular2-select/dist/select.component.js
```

`select.component.js` has console.log() outputs on lines: 54, 61, 62, 180, 181.

For example, in `ngOnInit` from `dist/select.component.js` there is an output:
```javascript
//...
    /** Event handlers. **/
    // Angular lifecycle hooks.
    SelectComponent.prototype.ngOnInit = function () {
        console.log('init');
        this.placeholderView = this.placeholder;
    };
//...
```

I tried to find *console.logs* in commits but I didn't find anything. I think they have been added after code generation in `dist/` folder for debugging purposes.

**Comment by basvandenberg** [comment]
Thanks for mentioning this! You are right, I last-minute double-checked some things by adding some `console.log`s, then reverted the git changes and forgot to rebuild before running `npm publish`... I published an updated version (1.0.0-beta.1) without the `console.log`s.

It makes a lot of sense not to have the `dist` directory in the git repository, but it makes running `npm publish` rather tricky. It's very easy to unintentionally publish the wrong distribution code. Maybe I should add a publish task to my gulp file that publishes after running a build.

**Comment by aboglioli** [comment]
Thank you!

Have you tried TravisCI? The build and publishing could be made with TravisCI easily.
